lib/matplotlib/backends/backend_pdf.py

```diff
@@ -883,13 +883,12 @@ def get_char_width(charcode):
             # Make the "Differences" array, sort the ccodes < 255 from
             # the multi-byte ccodes, and build the whole set of glyph ids
             # that we need from this font.
-            cmap = font.get_charmap()
             glyph_ids = []
             differences = []
             multi_byte_chars = set()
             for c in characters:
                 ccode = c
-                gind = cmap.get(ccode) or 0
+                gind = font.get_char_index(ccode)
```

Comment:
`get_char_index`, which calls `FT_Get_Char_Index` in freetype, returns 0 if the ccode doesn't not exist already, hence the removal of the `or 0`.